sites/public/cypress/support/commands.js

```diff
@@ -282,8 +285,8 @@ Cypress.Commands.add("step7AddHouseholdMembers", (application) => {
 })
 
 Cypress.Commands.add("step8PreferredUnits", (application) => {
-  application.preferredUnit.forEach((_, index) => {
-    cy.getByTestId("app-preferred-units").eq(index).check()
+  application.preferredUnitTypes.forEach((prefUnit) => {
```

Comment:
Previously we weren't actually checking the unit types we have in the Cypress mock